src/components/molecules/search-results-list/results-list-item.jsx

```diff
@@ -46,4 +46,4 @@ ResultsListItem.propTypes = {
 	result: PropTypes.object.isRequired,
 };
 
-export default ResultsListItem;
+export default ResultsListItem;
```

Comment:
run format on save!

Reply:
sorry - turned on in format on save in VScode (thought I did already)